minecraft_servers/lymmzy/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "lymmzy",
+  "nice_name": "LYMMZY",
+  "direct_ip": "lymmzy.com",
+  "server_wildcards": [
+    "%.lymmzy.com"
+  ],
+  "supported_languages": [
+     "en"
+  ],
+  "social": {
+    "discord": "https://discord.lymmzy.com"
```

Comment:
Please use the direct discord invite url (discord.com/invite/{invite}).
[Documentation reference](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url)